docs/_sources/developer/deployment.rst.txt

```diff
@@ -0,0 +1,163 @@
+Deployment
+----------
+
+This section contains a step-by-step guidance how to deploy this software.
+It`s based on an deployment of Django with Apache2 and mod_wsgi on a raspberry pi 4.
+
+Sources:
+~~~~~~~~
+
+* https://docs.djangoproject.com/en/3.0/howto/deployment/
+* https://wiki.ubuntuusers.de/Apache_2.4/
+* https://www.digitalocean.com/community/tutorials/how-to-serve-django-applications-with-apache-and-mod_wsgi-on-ubuntu-16-04
+
+Requirements:
+~~~~~~~~~~~~~
+
+* a web server
+* basic knowledge of bash
+
+Basic configuration:
+~~~~~~~~~~~~~~~~~~~~
+
+Update your distro
+
+  ``sudo apt update && sudo apt upgrade``
+
+Install Dependencies for the deployment. (We use Python3)
+
+  ``sudo apt-get install python3-pip apache2 libapache2-mod-wsgi-py3``
+
+You also need to install virtualenv, to seperate Python from your system's python
+
+  ``sudo pip install virtualenv``
+
+Now we need to clone the Git-Repository and setup the virtualenv for Python.
+First you need to change in the directory where you want to clone this Webapplication.
```

Comment:
First you need to change to the directory that you want to clone this web application to.